pinot-common/src/main/java/com/linkedin/pinot/common/request/helper/ControllerRequestBuilder.java

```diff
@@ -122,7 +124,18 @@ public static JSONObject buildCreateRealtimeTableJSON(String tableName, String s
         String timeColumnName, String timeType, String retentionTimeUnit, String retentionTimeValue, int numReplicas,
     String segmentAssignmentStrategy, JSONObject streamConfigs, String schemaName, String sortedColumn,
         List<String> invertedIndexColumns, String loadMode, boolean isHighLevel, List<String> noDictionaryColumns)
-    throws JSONException {
+      throws JSONException {
```

Comment:
Sorry, I didn't ask the question correctly. My concern was to just make sure that if there's no partition config, nothing gets written out. It seemed that with empty map, the code would have written out the empty partition config, as opposed to not writing it at all.